src/components/GoodsList.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+type Props = {
+  goods: string[];
+};
+
+export const GoodsList: React.FC<Props> = ({ goods }) => (
+  <ul>
+    {goods.map(good => (
+      <li data-cy="Good" key={good}>
```

Comment:
According to the checklist, when rendering a list, you should add a `key` prop that is unique for each element. If the `good` string is not guaranteed to be unique, you might encounter issues with React's rendering. Ensure that the `key` prop is a unique identifier for each list item. If `goods` can have duplicates, consider using the index of the map function or another unique identifier.